signale.js

```diff
@@ -26,11 +26,15 @@ class Signale {
     this._customTypes = Object.assign({}, options.types);
     this._disabled = options.disabled || false;
     this._scopeName = options.scope || '';
+    this._alignScopes = options.alignScopes || false;
     this._timers = options.timers || new Map();
     this._types = this._mergeTypes(defaultTypes, this._customTypes);
     this._stream = options.stream || process.stdout;
     this._longestLabel = defaultTypes.start.label.length;
 
+    this.constructor._scopeNames = this.constructor._scopeNames || [];
```

Comment:
Give the issue (#49) a read, as I go into more depth, but the TL;DR is: Scopes are defined per-instance. But in order to align the scopes, there needs to be some shared state that contains all the scopes, so that we can figure out which is the longest scope. We can create that shared state without having to re-design the interface(s) by using a static/class property. The name is sort of confusing, but `this.constructor` allows us to set a static/class property without having to explicit use the class name, e.g. `Signale._scopeNames =`.

Reply:
Now its pretty clear to me. Thanks for the explanation. :+1:

Reply:
No problem!